.github/workflows/dotnet.yml

```
@@ -0,0 +1,73 @@
+
+    # Get package version
+    - name: Get Package Version
+      if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+      run: |
+        $version = [system.diagnostics.fileversioninfo]::getversioninfo("bin\Release\netstandard2.0\Flow.Launcher.Localization.dll").productversion
+        echo "release_version=$version" | out-file -filepath $env:github_env -encoding utf-8 -append
+
+    # Publish to GitHub releases
+    - name: Publish GitHub releases
+      if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)